services/web/client/source/class/osparc/desktop/StudyEditor.js

```diff
@@ -376,6 +388,203 @@ qx.Class.define("osparc.desktop.StudyEditor", {
       }
     },
 
+    __getBrotherNodes: function(excludeNodeIds) {
+      const currentModel = this.__workbenchUI.getCurrentModel();
+      const workbench = this.getStudy().getWorkbench();
+
+      let brotherNodesObj = {};
+      if (currentModel.getNodeId) {
+        brotherNodesObj = currentModel.getInnerNodes(false);
+      } else {
+        brotherNodesObj = workbench.getNodes(false);
+      }
+
+      const brotherNodes = [];
+      for (const brotherNodeId in brotherNodesObj) {
```

Comment:
nice! the ``in`` operator!

Reply:
maybe it makes more sense to use ``let`` inside a loop.. although I guess it is working the same?